src/components/pages/observation/create/form/uploader/use-observation-resources.tsx

```diff
@@ -63,10 +64,6 @@ export const ObservationCreateProvider = (props: ObservationCreateContextProps)
   const fetchMyUploads = async () => {
     const { data } = await axListMyUploads();
 
-    if (!data.length) {
```

Comment:
Could pls mention y was the return logic removed

Reply:
I noticed that when indexedDB is out of sync and if all resources are removed from disk, this will prevent if from clearing. So this was removed